oxidation/bindings/README.md

```diff
@@ -19,24 +33,134 @@ npm run build  # Generate index.node (basically a .so that node can load)
+     cmdline-tools;9.0    | 9.0          | Android SDK Command-line Tools
+     cmdline-tools;latest | 9.0          | Android SDK Command-line Tools (latest)
+   ```
+
+6. Install `cmdline-tools`
+
+   ```shell
+   $TEMP_DIR/cmdline-tools/bin/sdkmanager --install "cmdline-tools;latest"
+   ```
+
+   > Note: using `latest` may not allow reproductible build on different dev env.
+   > But you can replace the value by another one (`0.9` for example).
+
+7. Update your `PATH` variable
+
+   ```shell
+   export PATH="$PATH:$ANDROID_HOME/cmdline-tools/latest/bin"
+   ```
 
-- Install Android Studio
-- In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Platform > Show Package Details` : Install Android SDK 30.0.3
-- In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Tools > Show Package Details` : Install Android NDK 22.1.7171670 and Android SDK Command-line Tools
-- Python command should be available in PATH environment variable (used by `rust-android-gradle` plugin)
+   > Note: the path is `latest/bin` because we install `cmdline-tools` a the tag `latest`.
+   > For `9.0`, it will be `9.0/bin` for example.
+
+8. Install Android `SDK-30.0.3` and `NDK-22.1.7171670` by running
+
+    ```shell
+    sdkmanager --install "ndk;<ndk version>" "build-tools;<build tool version>"
+    ```
+
+#### Install required rust targets
 
 Install Rust targets for cross compilation:
 
-```bash
-rustup target add armv7-linux-androideabi
-rustup target add i686-linux-android
-rustup target add aarch64-linux-android
-rustup target add x86_64-linux-android
+```shell
+rustup target add \
+  armv7-linux-androideabi \
+  i686-linux-android \
+  aarch64-linux-android \
+  x86_64-linux-android \
+```
+
+> The `bindings/android` directory is a valid Android project to generate a libparsec AAR.
+> However the AAR doesn't have to be built when using `../client/android` project (given it depends explicitly on the `bindings/android/libparsec`).
+
+#### Install Java development kit to build the application
+
+Install the latest LTS of `openjdk`:
+
+- Install the latest java LTS development kit (currently `JDK-17`).
+- If need to be, update your `JAVA_HOME` to point to the installed jdk version folder.
+
+### Update the Gradle dependencies locks
+
+```shell
+bash ./gradlew dependencies --write-locks
 ```
 
-*Note: The `bindings/android` directory is a valid Android project to generate a libparsec AAR.
-However the AAR doesn't have to be built when using `../client/android` project (given it depends explicitly on the `bindings/android/libparsec`).*
+> It you want to update the dependencies lock for a specific module use: `-p <module-path>`
+
+A more detailed documentation can be found here [Gradle - locking dependencies](https://docs.gradle.org/current/userguide/dependency_locking.html)
+
+### Update Gradle verification metadata
+
+```shell
+bash ./gradle --write-verification-metadata sha512 help
```

Comment:
any reason for using sha512 instead of 🌟 sha256 🌟 ? (the Gradle documentation uses sha256 in it example, sha256 is also used in poetry.lock/package.lock/Cargo.lock)

Reply:
I've the opportunity to use a more robust checksum algorithm, I take it.

Reply:
If sha256 was broken we wouldn't use it at all, but that's not the case 😅 
I don't think the theoretical improved robustness is worth the ~2x increase in size of `verification-metadata.xml` and (more importantly) the cognitive load of having the reader wonder why in this very peculiar case we use a different algorithm than everywhere in the codebase and than the official gradle documentation

Reply:
done